### ISSUE TITLE
Auto-type extended ASCII misses some characters in KDE/Qt applications

### ISSUE THREAD

**Issue by jensMF** [keepassxreboot/keepassxc#8976]
## Overview
When I use Auto-Type for passwords with extended ASCII characters in qTerminal, Konsole, Kate, Kwrite, KeePassXC, Kile and probaly other KDE/Qt applications, but also in VScodium and LibreOffice Calc, there are characters missing. When I use it with GNOME Terminal, Geany, moserial, firefox, or TexStudio and probably others it works correct.

## Steps to Reproduce
1.  generate a password with (enough) extended ASCII characters (I have a password length of 78, but with only extended ASCII 16 or 30 should be enough I guess)
2. open one of the affected programs (may be the problem exists even with other display-managers, I don't know, otherwise use plasma/KDE)
3. auto-type the password (with delay or without does not matter)

## Expected Behavior
The password is correctly typed with all characters and in correct order. 

## Actual Behavior
The password is typed, but some characters are missing.

## Context
The missing characters are not always the same. It differs depending on the delay and wether global auto-type is used or CTRL+SHIFT+V is used. But there are always missing characters, even with DELAY=500. It is possible that the same character is successfully typed before or after the position where it was not typed.

KeePassXC - Version 2.7.4
Revision: 63b2394

Qt 5.15.7

Betriebssystem: Manjaro Linux
CPU-Architektur: x86_64
Kernel: linux 6.0.15-1-MANJARO

Operating System: Linux
Desktop Env: KDE/plasmashell
Windowing System: X11

**Comment by droidmonkey** [comment]
Can you type them in by hand successfully? If it works in gnome then this is very unlikely to be our bug.

**Comment by jensMF** [comment]
Yes, I can of course enter them successfully by hand. Even KeePassXC can type them successfully. But sometimes it misses one and sometimes not.
I wrote that it works in GNOME Terminal, not in gnome. I did not test it in gnome because I have not installed gnome. I just have the "GNOME Terminal" application installed and there it works.
It seems to me, that it works with GTK applications but not with Qt applications (on my system).

**Comment by jensMF** [comment]
I tested an 8 character password with only extended ASCII and that works more reliable. It is successful every second time and when I wait for some time before I retry.

**Comment by droidmonkey** [comment]
If it's application/framework dependent then it's most likely not us. Are you using qwerty/us layout?

**Comment by jensMF** [comment]
No, I use qwerz/de layout.

**Comment by codekandis** [comment]
I can definitely confirm the issue. It occured the first time too long ago to remember the exact KPXC version I first experienced it. I missed to open an issue.

- usage of passwords with extended ASCII
- Debian LXDE (latest) - so I think it's gnome based (?)
- quertz/de layout
- terminal issues: no
- GUI related apps issues: yes

**Comment by droidmonkey** [comment]
This is probably a duplicate of the other keyboard layout issues

**Comment by sivaraam** [comment]
I could also confirm this issue. I have this when using Auto-type to fill passwords on Firefox (the extension fails to work for flatpak based apps). Let me know if anything is needed to debug this one.

**Comment by Chealer** [comment]
Thanks for reporting

Does this persist in 2.7.12 or ulterior? And if so:

1. Which version of LibreOffice do you use?
2. Can you provide a testcase which guarantees reproduction?
3. Which characters are missing?

Reproduction steps should be as deterministic as possible. Does this not reproduce with a given password?

<details>
  <summary>License</summary>
This comment and any other from myself on this ticket is offered under the terms of CC0 1.0.
</details>
